docs/arcaflow/workflows/expressions.md

```diff
@@ -260,6 +260,7 @@ Functions:
 | `toLower(string)`                                | string       | Returns the input with Unicode letters mapped to their lower case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
+| `readFile(string)`                               | string | Given a string containing a file path, returns the contents of the indicated file as a UTF-8 character string. When passing a relative file path, the path must be relative to the working directory from which the Arcaflow process is executed. Additionally, shell environment variables are not expanded.                                                                                     |
```

Comment:
I'd still prefer a pattern consistent with most of the others, like "Returns the contents of a file as a utf-8 character string". The second sentence might be more direct as something like "Relative paths are resolved from the Arcaflow process working directory". Additionally (😁), "Additionally" in the final sentence doesn't really help.

Reply:
I'll second Dave's second and third points.

As for the first point, I like consistency, but, since the input is not just a `string` but a file path, I think it behoves us to say something about the input to the function.  In many of the other functions, the characteristics of the input don't need much description because they are clear from the context, and so the "Given..." part of "the pattern" is elided...but I assert that it is still part of the pattern, even though it doesn't show up on the screen.  😉  But, I won't object if you take Dave's suggestion.